server/i18n/en.json

```diff
@@ -7307,8 +7307,32 @@
     "translation": "Unable to convert export to XML."
   },
   {
-    "id": "ent.actiance.export.write_file.appError",
-    "translation": "Unable to write the export file."
+    "id": "ent.actiance.export.writeExport.batch.create.temp.appError",
```

Comment:
Are any of these errors actually surfaced in the system console UI? If not, then we don't actually need to translate them. Only UI facing errors should be translated.

Reply:
Good point. I want to eventually surface errors in a much better way in the UI, so I'm trying to be strict and making them useful.

Reply:
Np. This was a request from the translation team to not increase their load unless necessary. So good to be be mindful of that.